mypy/meet.py

```diff
@@ -506,6 +506,17 @@ def visit_param_spec(self, t: ParamSpecType) -> ProperType:
         else:
             return self.default(self.s)
 
+    def visit_parameters(self, t: Parameters) -> ProperType:
+        # TODO: is this the right variance?
```

Comment:
I think it is correct, we need any instance of the return type to be compatible iwth both, which seems to be the intersection of their arguments e.g. `meet_types`.

Reply:
Also, if its possible to test these independently (I assume we can add tests in the semanal section that simply test this, if not thats a problem in itself) the "Add all the visitors" section of the diff should probably be migrated to a separate PR with its own semanal tests.